Update `ko` localized messages

Gudahtt opened the pull request:
These translations were provided by Lionbridge.

Gudahtt committed:
Update `ko` localized messages

These translations were provided by Lionbridge.

Gudahtt commented:
If any community contributors would like to help review this PR, it would be greatly appreciated!

I'll tag anyone here that has contributed in the past: @x86kernel @bowbowbow @nujabes403 @hackmod @neutiyoo 

I wanted to give you all the opportunity to provide feedback because we owe our current translations to you. Please don't feel obliged if you don't have the time to review, and please let me know if you'd like me to stop tagging you for reviews in the future.

Thanks!

neutiyoo commented on app/_locales/ko/messages.json:
```diff
@@ -503,84 +803,128 @@
+    "message": "잠금"
+  },
+  "lockTimeTooGreat": {
+    "message": "자금 시간이 너무 김"
```
'자금' -> '잠금'

Gudahtt replied:
Thanks, updated!
```suggestion
    "message": "잠금 시간이 너무 김"
```

nujabes403 commented:
LGTM

Gudahtt committed:
Update `lockoutTimeTooGreat` message

neutiyoo commented on app/_locales/ko/messages.json:
```diff
@@ -592,206 +936,318 @@
-    "message": "메타마스크 버전"
+    "message": "MetaMask 버전"
+  },
+  "metametricsCommitmentsAllowOptOut": {
+    "message": "언제든 설정을 통해 옵트아웃할 수 있습니다."
+  },
+  "metametricsCommitmentsBoldNever": {
+    "message": "안 함",
+    "description": "This string is localized separately from some of the commitments so that we can bold it"
+  },
+  "metametricsCommitmentsIntro": {
+    "message": "MetaMask에서는.."
+  },
+  "metametricsCommitmentsNeverCollectIP": {
+    "message": "$1은(는) 전체 IP 주소를 수집하지 않습니다.",
+    "description": "The $1 is the bolded word 'Never', from 'metametricsCommitmentsBoldNever'"
+  },
+  "metametricsCommitmentsNeverCollectKeysEtc": {
+    "message": "$1은(는) 키, 주소, 거래, 잔액, 해시 또는 개인 정보를 수집합니다.",
+    "description": "The $1 is the bolded word 'Never', from 'metametricsCommitmentsBoldNever'"
+  },
+  "metametricsCommitmentsNeverSellDataForProfit": {
+    "message": "$1은(는) 수익을 위해 데이터를 판매합니다. 절대로요!",
+    "description": "The $1 is the bolded word 'Never', from 'metametricsCommitmentsBoldNever'"
+  },
```
We are replacing `$1` with `Never` which is the value from `metametricsCommitmentsBoldNever`.
For example, `"$1 collect your full IP address"` will be `"Never collect your full IP address"`. This makes sense.

However, in Korean, we are replacing `$1` with `안 함` which is the value from `metametricsCommitmentsBoldNever`.

a) `$1은(는) 전체 IP 주소를 수집하지 않습니다.`
b) `안 함은(는) 전체 IP 주소를 수집하지 않습니다.`

and the a) will be the b) after `$1` is replaced with `안 함`.

In the case, the sentence won't make sense as I see it.

A possible solution can be the following:

we set `$1` as `하지 않습니다` and change the related ones as the following

- `전체 IP 주소를 수집$1.` (`전체 IP 주소를 수집하지 않습니다.`)
- `키, 주소, 거래, 잔액, 해시 또는 개인 정보를 수집$1.` (`키, 주소, 거래, 잔액, 해시 또는 개인 정보를 수집하지 않습니다.`)
- `수익을 위해 데이터를 판매$1.` (`수익을 위해 데이터를 판매하지 않습니다.`)

neutiyoo commented on app/_locales/ko/messages.json:
```diff
@@ -592,206 +936,318 @@
+  "metametricsCommitmentsAllowOptOut": {
+    "message": "언제든 설정을 통해 옵트아웃할 수 있습니다."
+  },
+  "metametricsCommitmentsBoldNever": {
+    "message": "안 함",
+    "description": "This string is localized separately from some of the commitments so that we can bold it"
+  },
+  "metametricsCommitmentsIntro": {
+    "message": "MetaMask에서는.."
+  },
+  "metametricsCommitmentsNeverCollectIP": {
+    "message": "$1은(는) 전체 IP 주소를 수집하지 않습니다.",
+    "description": "The $1 is the bolded word 'Never', from 'metametricsCommitmentsBoldNever'"
+  },
+  "metametricsCommitmentsNeverCollectKeysEtc": {
+    "message": "$1은(는) 키, 주소, 거래, 잔액, 해시 또는 개인 정보를 수집합니다.",
+    "description": "The $1 is the bolded word 'Never', from 'metametricsCommitmentsBoldNever'"
+  },
+  "metametricsCommitmentsNeverSellDataForProfit": {
+    "message": "$1은(는) 수익을 위해 데이터를 판매합니다. 절대로요!",
+    "description": "The $1 is the bolded word 'Never', from 'metametricsCommitmentsBoldNever'"
+  },
```
## Background

We are replacing `$1` with `Never` which is the value from `metametricsCommitmentsBoldNever`.
For example, `"$1 collect your full IP address"` will be `"Never collect your full IP address"`. This makes sense.

## Problem
However, in Korean localization, we are replacing `$1` with `안 함` which is the value from `metametricsCommitmentsBoldNever`.

a) `$1은(는) 전체 IP 주소를 수집하지 않습니다.`
b) `안 함은(는) 전체 IP 주소를 수집하지 않습니다.`

The a) will be the b) after `$1` is replaced with `안 함`.
In the case, the sentence won't make sense as I see it.

## Solution
Therefore, the following should be changed:
- `metametricsCommitmentsBoldNever`
- `metametricsCommitmentsNeverCollectIP`
- `metametricsCommitmentsNeverCollectKeysEtc`
- `metametricsCommitmentsNeverSellDataForProfit`

A possible solution can be the following:

we can set `$1` as `하지 않습니다` and change the related ones as the following:

- `전체 IP 주소를 수집$1.` (`전체 IP 주소를 수집하지 않습니다.`)
- `키, 주소, 거래, 잔액, 해시 또는 개인 정보를 수집$1.` (`키, 주소, 거래, 잔액, 해시 또는 개인 정보를 수집하지 않습니다.`)
- `수익을 위해 데이터를 판매$1.` (`수익을 위해 데이터를 판매하지 않습니다.`)

Gudahtt replied:
Great catch!

Either one of those options seem fine to me :thinking: Which do you think would be more idiomatic/commonplace?

neutiyoo replied:
The second one looks good to me.

neutiyoo replied:
I realized that solution B and C do not emphasize the negation so I came up with a A solution.
Unlike B or C, A emphasizes the negation with `절대` and is more close to the original one.

Therefore, the solution A is the more idiomatic one IMO.

neutiyoo replied:
I realized that the previous solutions I mentioned(solution B and C) do not emphasize the negation.

Hence, I came up with the solution **A**.
Unlike B or C, A emphasizes the negation with `절대` making the meaning more close to the original one.

Therefore, the solution A is the correct and idiomatic one IMO.

Gudahtt replied:
Great, thanks!

I have applied solution A in 0cb1ae01a530bd1508540ed02a0a2c96d115e7ea - let me know if I've made a mistake anywhere.

Gudahtt committed:
Update statements with bold 'Never'

The sentences did not make sense previously. They have been updated to
be grammatically correct and to emphasize the negation.

Gudahtt committed:
Apply suggested change to 'swapNewQuoteIn'

Gudahtt commented on app/_locales/ko/messages.json:
```diff
@@ -989,118 +1489,433 @@
+  "swapNewQuoteIn": {
+    "message": "$1의 새 견적",
+    "description": "Tells the user the amount of time until the currently displayed quotes are update. $1 is a time that is counting down from 1:00 to 0:00"
+  },
```
Thanks! I've applied the first solution you suggested here: 71a9f57c696db63891bc7f710696f284f6ee9afa

Please let me know if I've made a mistake, or if you think solution 2 would be more appropriate.

neutiyoo reviewed (dismissed):
LGTM

Gudahtt commented:
This PR has been rebased onto `develop` before merging to ensure that the `yourSigRequested` message was not present (it was removed in #10300).

Thanks again for the review!
